lib/Selenium/Remote/Driver.pm

```diff
@@ -144,6 +144,7 @@ you please.
         'javascript'           - <boolean>  - whether javascript should be supported
         'accept_ssl_certs'     - <boolean>  - whether SSL certs should be accepted, default is true.
         'firefox_profile'      - Profile    - Use Selenium::Firefox::Profile to create a Firefox profile for the browser to use
+        'marionette_enabled'   - <boolean>  - whether Firefox should enable Marionette. default if false
```

Comment:
Ah! I thought these were all the available options, that's why I added it there. No I haven't ever used the selenium standalone server.

I'll probably remove it from the docs here.